apps/balancer-tools/src/app/apr/pool/[network]/[poolId]/page.tsx

```diff
@@ -2,12 +2,17 @@ import { Suspense } from "react";
 
 import HistoricalAPRChart from "#/app/apr/pool/(components)/HistoricalAPRChart";
 import PoolOverviewCards from "#/app/apr/pool/(components)/PoolOverviewCards";
+import Breadcrumb from "#/app/apr/round/(components)/Breadcrumb";
 import { Spinner } from "#/components/Spinner";
 
-export default async function Page({ params }: { params: { poolId: string } }) {
-  const { poolId } = params;
+export default async function Page({
+  params: { roundId, network, poolId },
+}: {
+  params: { roundId: string; poolId: string; network: string };
+}) {
   return (
     <div className="flex flex-1 h-full w-full flex-col justify-start rounded-3xl text-white gap-y-3">
+      <Breadcrumb network={network} poolId={poolId} roundId={roundId} />
```

Comment:
as `Breadcrumb` is already a client component we could call `useParams` instead of having to prop drill. Just to follow the pattern I did on https://github.com/bleu-studio/balancer-tools/pull/246/commits/8357aabe831c6875cc6b084dbcd228a9fb87dc37

If the decision is to maintain this structure, I would refactor  `HeaderEndButton`

Reply:
I can't believe I didn't realised I could just that, and went the hardest route. Thanks for the reminder!